lib/sidekiq/symbols.rb

```diff
@@ -2,31 +2,44 @@
 
 module Sidekiq
 module Symbols
+  def self.symbolize_args(args)
+    symbolize_keys = lambda do |arg|
+      h = {}
+      arg.each do |k, v|
+        k = k.to_sym if k.respond_to?(:to_sym)
+        h[k] = v.is_a?(Hash) ? symbolize_keys.call(v) : v
+      end
+      h
+    end
+
+    symbolized_args = args.map do |arg|
+      if arg.is_a?(Hash)
+        symbolize_keys.call(arg)
+      else
+        arg
+      end
+    end.to_a
+  end
+
   def self.included(klass)
     klass.class_eval { prepend(Symbolizer) }
+
+    klass.class_eval do
+      def self.inherited(subclass)
```

Comment:
I suppose a workaround is to add even more magic by using `prepend` to wrap the `self.inherited`? :mindblown: